### DISCUSSION TITLE
Disable automatic pull requests creation for dependabot

### DISCUSSION THREAD

**Discussion by xvitaly** [community/community#21129] [Actions]
Add the ability to disable automatic generation of pull requests for dependabot and use issues instead.

I want to see alerts as issues, because dependabot is [broken for NuGet](https://github.com/dependabot/dependabot-core/issues/1422) for ages.

**Comment by stavares843** [comment]
seems there's an open issue on their repo [here](https://github.com/dependabot/dependabot-core/issues/2242) 🔨
